cleverhans/attacks.py

```diff
@@ -31,14 +31,8 @@ def __init__(self, model, back='tf', sess=None):
             sess = tf.get_default_session()
 
         if not isinstance(model, Model):
-            if hasattr(model, '__call__'):
-                warnings.warn("CleverHans support for supplying a callable"
-                              " instead of an instance of the"
-                              " cleverhans.model.Model class is"
-                              " deprecated and will be dropped on 2018-01-11.")
-            else:
-                raise ValueError("The model argument should be an instance of"
-                                 " the cleverhans.model.Model class.")
+            raise ValueError("The model argument should be an instance of"
```

Comment:
This error is thrown by the tests because the model was defined as a callable and is not wrapped automatically anymore: eg., https://github.com/tensorflow/cleverhans/blob/master/tests_tf/test_attacks.py#L47

Reply:
Pushed a bunch of fixes to the tests